src/lib/skilling/skills/herblore/mixables/potions.ts

```diff
@@ -400,6 +400,15 @@ const Potions: Mixable[] = [
 		tickRate: 2,
 		bankTimePerPotion: 0.5
 	},
+	{
+		item: getOSItem('Forgotten brew (4)'),
+		aliases: ['forgotten brew (4)', 'forgotten brew', 'forgotten'],
+		level: 91,
+		xp: 145,
+		inputItems: new Bank({ 'Ancient brew (4)': 1, 'Ancient essence': 80 }),
```

Comment:
You dont have to change it, but for future reference, try not use this format for making banks anymore. do `new Bank().add().add()`